packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -470,6 +461,15 @@ export default mixins(showMessage, nodeHelpers).extend({
 		isSharingAvailable(): boolean {
 			return this.settingsStore.isEnterpriseFeatureEnabled(EnterpriseEditionFeature.Sharing);
 		},
+		getDefaultCredentialTypeName(): string {
```

Comment:
nitpick - computed property so does not require `get` since it's not a function

Reply:
Agree, updated.